en/user_guide/transverse.rst

```diff
@@ -0,0 +1,27 @@
+Global page or transverse node
+==============================
+
+Each singular website has a global page (node transverse) per languages, it is a node containing blocks common to the other pages.
+This node allows to configure blocks, those blocks which would be available in all the other pages of this website.
+
+Global pages have several languages, don't have a `workflow`_ and have only one version.
+
+Transverse block
+----------------
+
+Transverse blocks are blocks in the area of the transverse node, these blocks are common to the other pages.
```

Comment:
This sentence is rephrasing what you defined earlier.
Instead you should simply say that Transverse blocks are blocks defined in transverse node.